libs/dh/esett/feature-metering-gridarea-imbalance/src/lib/dh-metering-gridarea-imbalance.component.ts

```diff
@@ -101,9 +98,15 @@ export class DhMeteringGridAreaImbalanceComponent implements OnInit {
   });
   totalCount = 0;
 
-  pageMetaData$ = this.store.pageMetaData$;
-  sortMetaData$ = this.store.sortMetaData$;
-  filters$ = this.store.filters$;
+  pageMetaData = toSignal(this.store.pageMetaData$, {
+    initialValue: { pageIndex: 0, pageSize: 100 },
+  });
+  sortMetaData = toSignal(this.store.sortMetaData$, {
+    initialValue: { active: 'receivedDateTime', direction: 'desc' },
+  });
+  filters = toSignal(this.store.filters$, {
+    initialValue: { valuesToInclude: MeteringGridImbalanceValuesToInclude.Imbalances },
+  });
```

Comment:
question: could we use `requireSync: true` instead, since these initial values are already provided in the store configuration? if not, maybe consider importing these values from the store instead of duplicating it here.